README.md

```diff
@@ -17,8 +17,8 @@ Bugs should be filed under the Issues section of this repo.
 Feature requests can be submitted at https://feedback.mongodb.com/forums/924145-atlas - just select the Terraform plugin as the category or vote for an already suggested feature.
 
 ## Requirements
-- [Terraform](https://www.terraform.io/downloads.html) 1.0+
-- [Go](https://golang.org/doc/install) 1.21 (to build the provider plugin)
+- [Terraform Version](https://www.terraform.io/downloads.html) Minimum of 1.0 (as we have migrated to [Plugin Protocol Version 6](https://developer.hashicorp.com/terraform/plugin/terraform-plugin-protocol#protocol-version-6)), however for your saftey we strongly recondmend only consuming versions of Terraform that are currently receiving Security/Maintenance Updates. For latest supported versions see: https://endoflife.date/terraform 
```

Comment:
```suggestion
- [Terraform Version](https://www.terraform.io/downloads.html) Minimum of 1.0 (as we have migrated to [Plugin Protocol Version 6](https://developer.hashicorp.com/terraform/plugin/terraform-plugin-protocol#protocol-version-6)), however for your saftey we strongly recommend only consuming versions of Terraform that are currently receiving Security/Maintenance Updates. For latest supported versions see: https://endoflife.date/terraform 
```

Reply:
I wonder if we should change this and stop putting a number like `minimum of 1.0`. What if we said that we only actively support the supported terraform versions? This way:
* it becomes a semantic that will never need to be updated
* it is stricter yes but it doesn't mean it won't work for previous versions, it just means we don't actively support any bugs with such combinations
* if customers are particularly important yes, we can make exceptions

Then we can add a "Why" section and you can explain that it's for security reasons like you said.

Reply:
Updated as discussed off sync, also just FYI will update again to move minimum version to HashiCorp Terraform v1.3+ in Q2 2024 (this is when several large customers are planning to migrate off Terraform v1.0+).